<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(6, 75, 5);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>40</y>
      <width>701</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>               ID                                CODIGO                      PRODUTO                          PREÃ‡O                     CATEGORIA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ld_id">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>139</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ld_codigo">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>90</y>
      <width>139</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ld_produto">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>90</y>
      <width>139</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ld_preco">
    <property name="geometry">
     <rect>
      <x>438</x>
      <y>90</y>
      <width>139</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ld_categoria">
    <property name="geometry">
     <rect>
      <x>581</x>
      <y>90</y>
      <width>139</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalvarEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);</string>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>-9</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Edite este produto</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
